src/app/ReadHandler.cpp

```diff
@@ -848,5 +854,14 @@ void ReadHandler::SetDirty(const AttributePathParams & aAttributeChanged)
         mAttributeEncoderState = AttributeValueEncoder::AttributeEncodeState();
     }
 }
+
+Transport::SecureSession * ReadHandler::GetSession() const
```

Comment:
Why does this reduce size of ReadHandler?

Reply:
Updated

```
Reorder the fields in reduce inner padding
Pack all (exactly 8) boolean variables into BitFlags
Remove Messaging::ExchangeManager * mpExchangeMgr since it is always the same as the one in interaction model engine.
Remove mInitiatorNodeId and mSubjectDescriptor since we can get them from SessionHandler and there default values are also invalid values.
Fix the type of SubscriptionId, which should be uint32_t
```